elementor/elementor
# Tweak: Synchronous elements initialization may produce long JavaScript tasks [ED-8112]

## PR Checklist

## PR Type
What kind of change does this PR introduce?


## Summary

This PR fixes issue, described in #15228 which may cause browser's main thread to be blocked for some time in a case of long Elementor's initialization process.

This PR can be summarized in the following changelog entry:

* Improve browser responsiveness during Elementor's initialization process

## Description

This PR provides fix, explained into #15228 which enforces initialization of every page element to be done into separate main thread event cycle which effectively breaks code into smaller chunks, reducing chance for long JavaScript tasks to be run.

## Test instructions
This PR can be tested by following these steps:

1. Put multiple (10+) elements of different types that will require JavaScript initialization on the page
2. Open page in browser which allows capturing performance metrics
3. Open "Performance" tab in browser's DevTools
4. Measure page loading performance by running performance information gathering
5. Take a look at "Main" line at a time of DCL event, ensure that there is no, or much less blocking time of browser's main thread.

## Quality assurance


Fixes #15228
